src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -63,15 +63,15 @@ trait ClusterComponent extends LeoComponent {
     def workerDiskSize =              column[Option[Int]]       ("workerDiskSize")
     def numberOfWorkerLocalSSDs =     column[Option[Int]]       ("numberOfWorkerLocalSSDs")
     def numberOfPreemptibleWorkers =  column[Option[Int]]       ("numberOfPreemptibleWorkers")
-    def operationName =               column[String]            ("operationName",         O.Length(254))
+    def operationName =               column[Option[String]]    ("operationName",         O.Length(254))
     def status =                      column[String]            ("status",                O.Length(254))
     def hostIp =                      column[Option[String]]    ("hostIp",                O.Length(254))
     def creator =                     column[String]            ("creator",               O.Length(254))
     def createdDate =                 column[Timestamp]         ("createdDate",           O.SqlType("TIMESTAMP(6)"))
     def destroyedDate =               column[Timestamp]         ("destroyedDate",         O.SqlType("TIMESTAMP(6)"))
```

Comment:
Not your change, but below on the line:
```
def uniqueKey = index("IDX_CLUSTER_UNIQUE", (googleProject, clusterName), unique = true)
```
We might want to make this `(googleProject, clusterName, destroyedDate)` to match the database schema.